R/explorespec.R

```diff
@@ -101,7 +101,7 @@ explorespec <- function(rspecdata, by = NULL,
   arg$x <- wl
 
   if (is.null(arg$col)) {
-    col_list <- viridisLite::viridis(max(table(by)))
+    col_list <- palette.colors(n = 8, palette = "Okabe-Ito", alpha = 1)
```

Comment:
Review note (so that I remember): there's no point setting `n` to more than 8 since the palette only has 8 colours. On the other hand, it doesn't hurt to set it as high as 8.